classes/submissionFile/SubmissionFile.inc.php

```diff
@@ -13,16 +13,16 @@
  * @brief Submission file class.
  */
 
-namespace PKP\submission;
+namespace PKP\submissionFile;
 
 use APP\i18n\AppLocale;
-
-use PKP\db\DAORegistry;
+use PKP\core\PKPApplication;
 
 // Define the file stage identifiers.
 
 class SubmissionFile extends \PKP\core\DataObject
 {
+    public const SUBMISSION_FILE_PUBLIC = 1;
```

Comment:
I think this is deprecated and shouldn't be used any more.

Reply:
I've found it being used on classes/install/Upgrade.inc.php:541, so, named it as a constant, just for tracking.

Reply:
https://github.com/pkp/ojs/pull/3160/files#diff-590dda691fda77f1799f0b82b40112f4b5890c543ba9fb61de436af7a9557220R541

Reply:
It used to be defined as a constant, but it was removed when it became historical only. Since it's only used in the legacy upgrade code, and eventually will be removed from there, it's best to keep it there only.

Reply:
This still needs fixing.